mne/io/snirf/tests/test_snirf.py

```diff
@@ -57,7 +57,9 @@
 )
 
 # Kernel
-kernel_hb = testing_path / "SNIRF" / "Kernel" / "Flow50" / "Portal_2021_11" / "hb.snirf"
+kernel_path = testing_path / "SNIRF" / "Kernel" / "Flow50" / "Portal_2021_11"
```

Comment:
Here are some files to add. The path could be updated if it matters, these are no longer Flow50/Flow1 but Flow2, and the version is latest (aka, Portal_2024-10-23).

- Type 201 (Time Domain Gated): [c345d04_2.snirf](https://drive.google.com/file/d/1OMCtRcQweOOZ295jQk0ymeqwxWG-hEMc/view?usp=drive_link) (there is no current equivalent)
- Type 301 (Time Domain Moments): [c345d04_3.snirf](https://drive.google.com/file/d/1XdcKDx3lt6cdS0hFQsDDDVhCtvjoQh10/view?usp=drive_link) (replace `td_moments.snirf`)
- Type 99999 (Time Domain Hb Moments): [c345d04_5.snirf](https://drive.google.com/file/d/13mvz96h4iiIXr0mGgIAHQ4MyC6a54bPQ/view?usp=drive_link) (replace `hb.snirf`)